### PR TITLE
Update ScaleIO volume plugin default readOnly value

### PR DESCRIPTION
**Pull request by vladimirvivien**
This commit updates the code to set readOnly attribute to be set to false.

**What this PR does / why we need it**:
This PR is a minor fix that updates the default value of `readOnly` attribute to `false`.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rootfs** [comment]
@vladimirvivien redirect the PR against master branch

**Comment by vladimirvivien** [comment]
@rootfs I am trying to get this in 1.6 do I still base against master ?  What's the workflow to get changes before code freeze end ?

**Comment by rootfs** [comment]
Open a PR to master. Once merged and not in release-1.6, then request a cherry pick.

**Comment by vladimirvivien** [comment]
@rootfs thanks for the info.

**Comment by vladimirvivien** [comment]
@k8s-bot ok to test

**Comment by rootfs** [comment]
@k8s-bot ok to test

**Comment by vladimirvivien** [comment]
@k8s-bot gce etcd3 e2e test this

File: examples/volumes/scaleio/README.md [hunk 1]
```diff
@@ -80,7 +80,20 @@ $ kubectl create -f ./examples/volumes/scaleio/secret.yaml
...
+|-----------|-------------|
+| gateway | address to a ScaleIO API gateway (required)|
+| system  | the name of the ScaleIO system (required)|
+| protectionDomain| the name of the ScaleIO protection domain (default `default`)|
+| storagePool| the name of the volume storage pool (default `default`)|
+| storageMode| the storage provision mode: `ThinProvisionned` (default) or `ThickProvisionned`|
+| volumeName| the name of an existing volume in ScaleIO (required)|
+| secretRef:name| reference to a configuered Secret object (required, see Secret earlier)|
+| readOnly| specifies the access mode to the mounted volume (default `false`)|
+| fsType| the file system to use for the volume (default `xfs`)|
```
Comment (rootfs) [inline examples/volumes/scaleio/README.md]: (also for rest for the patch) this is inconsistent with api doc https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L1091.

Comment (vladimirvivien) [inline examples/volumes/scaleio/README.md]: @rootfs Hi, I am not sure what you mean that desc for  `fsType` is inconsistent with the api doc?  Are you saying I should copy what's in the API doc in the examples doc ?

**Comment by vladimirvivien** [comment]
@rootfs can I get a LGTM on this PR so it can be cherry picked for 1.6 ?

File: examples/volumes/scaleio/README.md (hunk 1 above)
Comment (rootfs) [inline examples/volumes/scaleio/README.md]: yes, please make sure the API doc is consistent with README.

**Comment by rootfs** [comment]
@vladimirvivien please address the comment and ping me back.

File: examples/volumes/scaleio/README.md (hunk 1 above)
Comment (vladimirvivien) [inline examples/volumes/scaleio/README.md]: @rootfs the doc and the api is sync'd

**Comment by vladimirvivien** [comment]
@rootfs ready for another look for a LGTM.

**Comment by rootfs** [comment]
@vladimirvivien need @thockin approve to get it merged

**Comment by saad-ali** [comment]
Doc fixes ok for 1.6 post-code freeze merge

**Comment by saad-ali** [comment]
Minor isolated bug fix should be ok for 1.6 post-code freeze merge.

**Comment by vladimirvivien** [comment]
Thank you @rootfs and @saad-ali

**Pull request merged**